docs/modelserving/observability/grafana_dashboards.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Dashboards 
+
+Some example Grafana dashboards are available in GrafanaLabs. 
+
+## Knative HTTP Dashboard (if using serverless mode)
+
+The [Knative HTTP Grafana dasbhoard](https://grafana.com/grafana/dashboards/18032-knative-serving-revision-http-requests/) was built from [Knative's sandbox monitoring example]https://github.com/knative-sandbox/monitoring().
+
+## KServe ModelServer Latency Dashboard
+
+A template dashboard for [KServe ModelServer Latency](https://grafana.com/grafana/dashboards/17969-kserve-modelserver-latency/) contains example queries using the prometheus metrics for pre/post-process, predict and explain in milliseconds. The query is a [histogram quantile](https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile). A fifth graph shows the total number of requests to the predict endpoint. This graph covers all KServe's ModelServer runtimes - lgbserver, paddleserver, pmmlserver, sklearnserver, xgbserver, custom transformer/predictor.
+
+## KServe TorchServe Latency Dashboard
+
+A template dashboard for [KServe TorchServe Latency](https://grafana.com/grafana/dashboards/18026-kserve-torchserve-latency/) contains an inference latency graph which plots the [rate](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) of the TorchServe metric `ts_inference_latency_microseconds` in milliseconds. A second graph plots the rate of TorchServe's internal queue latency metric `ts_queue_latency_microseconds` in milliseconds. A third graph plots the total requests to the TorchServe Inference Service. 
+
+## KServe Triton Latency Dashboard 
+
+A template dashboard for [KServe Triton Latency](https://grafana.com/grafana/dashboards/18027-kserve-triton-latency/) contains five latency graphs with the rate of Triton's input (preprocess), infer (predict), output (postprocess), internal queue and total latency metrics plotted in milliseconds. Triton outputs metrics on GPU usage as well, and the template plots a gauge of the percentage of GPU memory usage in bytes.
```

Comment:
Add link to the triton inference server metrics reference 
https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md

Reply:
Ok I have on other page but will be useful here too